client/my-sites/hosting/main.js

```diff
@@ -69,11 +69,15 @@ class Hosting extends Component {
 	}
 
 	componentDidMount() {
+		this.props.fetchAutomatedTransferStatus( this.props.siteId );
```

Comment:
The problem with calling this method is that it duplicates polling when transferring by clicking "Activate" and not reloading the page.

In this scenario, [`initiateThemeTransfer`](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/themes/actions/theme-transfer.js#L56) and [`receiveStatus`](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/data-layer/wpcom/sites/automated-transfer/status/index.js#LL25C1-L25C1) are polling the back-end (which duplicates the number of necessary requests).

But that's not all: _both_ are dispatching tracks events whenever the transfer gets completed. This is not a bad thing per se, but in our situation, it's not desired. We should find a way of not dispatching duplicated Tracks events only in this scenario.

Reply:
> In this scenario, [initiateThemeTransfer](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/themes/actions/theme-transfer.js#L56) and [receiveStatus](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/data-layer/wpcom/sites/automated-transfer/status/index.js#LL25C1-L25C1) are polling the back-end (which duplicates the number of necessary requests).

Yup! both those actions should be using the same polling mechanism to DRY out the code.

Reply:
I commented on Slack too. I think the problem comes from us using [initiateThemeTransfer](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/themes/actions/theme-transfer.js#L56) to check for an Atomic transfer. IMO, It would be better to have an `initiateAtomicTransfer` and a new endpoint that could be polled for Atomic site transfers.

Reply:
This will attempt to fetch even for simple sites, I guess that might not be an issue but it's something I noticed while working on the `/sites` transfer status loader.